packages/peregrine/lib/talons/Wishlist/Wishlist/useWishlist.ce.js

```diff
@@ -60,8 +60,8 @@ export const useWishlist = props => {
             console.error(err);
 
             // Make sure any errors from the mutation are displayed.
-            if (setDisplayError) {
-                setDisplayError(true);
+            if (onWishlistUpdateError) {
+                onWishlistUpdateError(true);
```

Comment:
May as well pass the error back.

Reply:
Yeah, that was the plan, might have missed this. Thanks, dude.